tests/flow/tests_pytorch.py

```diff
@@ -284,6 +284,7 @@ def run():
         exception = e
         env.assertEqual(type(exception), redis.exceptions.ResponseError)
         env.assertEqual("Model did not generate the expected batch size", exception.__str__())
+    t.join()
```

Comment:
I would add "env.assertTrue(False)" after lines 274 and 282, otherwise the test doesn't guarantee that those exceptions were raised...

Reply:
```python
ret = con.execute_command('AI.MODELRUN', 'm{1}', 'INPUTS', 'd{1}', 'e{1}', 'OUTPUTS', 'f1{1}', 'f2{1}')
env.assertEqual(ret, b'OK')
```
as a single call might succeed, depends on the threads timing